CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ the endpoint has been removed. Use the `/metrics` endpoint to collect system sta
 1. [20819](https://github.com/influxdata/influxdb/pull/20819): Fix Single Stat graphs with thresholds crashing on negative values.
 1. [20809](https://github.com/influxdata/influxdb/pull/20809): Fix InfluxDB port in Flux function UI examples. Thanks @sunjincheng121!
 1. [20827](https://github.com/influxdata/influxdb/pull/20827): Remove unauthenticated, unsupported `/debug/vars` HTTP endpoint.
+1. [20856](https://github.com/influxdata/influxdb/pull/20856): Respect 24 hour clock formats in the UI and allow more choices
```

Comment:
Nitpick question: should this be a feature instead of a bug fix?

Reply:
No, it's a bug fix. We've always intended to have the 24 hour clock but it was not properly rendered.

Reply:
... and the additional choices is very minor. In case a user had gotten used to the 12 hour clock by picking one of the 24 clock options previously, they can still keep that format by selecting one of the new choices.